micro-rdk/src/common/conn/server.rs

```diff
@@ -95,11 +95,12 @@ pub struct ViamServerBuilder<M, C, T, NetworkType, CC = WebRtcNoOp, D = WebRtcNo
     _marker: PhantomData<T>,
     exec: Executor,
     app_connector: C,
-    app_config: AppClientConfig,
+    robot_credentials: RobotCredentials,
     max_connections: usize,
     app_client_tasks: Vec<Box<dyn PeriodicAppClientTask>>,
     network: NetworkType,
     app_client: Option<AppClient>,
+    rpc_host: String,
```

Comment:
Now that we have a clearer understanding of what this field is for, can we improve its name?

Reply:
I want to keep it this way because it matches the corresponding header name on the gRPC request, would you settle for a comment?

Reply:
No, that's fine then if it is following the protos. I don't think it needs a comment.